python/ray/dashboard/subprocesses/module.py

```diff
@@ -238,7 +240,7 @@ def run_module(
     cls: type[SubprocessModule],
     config: SubprocessModuleConfig,
     incarnation: int,
-    ready_event: multiprocessing.Event,
+    conn_to_parent: multiprocessing.connection.Connection,
```

Comment:
let's make the naming consistent. Either `conn_to_parent & conn_to_child` or `parent_conn & child_conn`.